tests/drum/utils.py

```diff
@@ -0,0 +1,236 @@
+import os
```

Comment:
creating utils files is frowned upon, please try to categorize this into specific utilities and name correspondingly.

Reply:
split into utils and server utils